src/content/developers/tutorials/waffle-test-simple-smart-contract/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Testing simple smart contract with Waffle library
+description: Tutorial for beginners
+author: Ewa Kowalska
+tags: ["smart contracts", "solidity", "Waffle", "testing"]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-26-02
+---
+## In this tutorial you'll learn how to
+- Test the changes of wallet balance
+- Test emission of events with specified arguments
+- Assert that a transaction was reverted
+
+## Assumptions
+- You can create a new JavaScript or TypeScript project
+- You have some basic experience with tests in Javascript
+- You have used some package managers like yarn or npm
+- You possess very basic knowledge of smart contracts and Solidity
+
+# Getting started
+The tutorial demonstrates test setup and run using yarn, but there is no problem if you prefer npm - I will provide proper references to the official Waffle [documentation](https://ethereum-waffle.readthedocs.io/en/latest/index.html). 
+## Install Waffle
+[Add](https://ethereum-waffle.readthedocs.io/en/latest/getting-started.html#installation) ethereum-waffle package to the dev dependencies of your project.
+```bash
+yarn add --dev ethereum-waffle
+```
+## Example smart contract
+
+During the tutorial we'll work on a simple smart contract example - EtherSplitter. It does not much apart from allowing anyone to send some Wei and split it evenly between two predefined receivers.\
+The split function requires the number of Wei to be even, otherwise it will revert. For both receivers it performs a Wei transfer followed by emission of the Transfer event.
```

Comment:
Been using convention of lowercase "wei" across the site (referencing the whitepaper)